packages/rum/src/domain/requestCollection.ts

```diff
@@ -109,7 +109,7 @@ export function trackFetch(lifeCycle: LifeCycle, configuration: Configuration, t
         spanId: context.spanId,
         startTime: context.startTime,
         status: context.status,
-        traceId: context.traceId,
+        traceId: context.status === 0 ? undefined : context.traceId,
```

Comment:
What about having this logic in `tracer` since it is related to tracing?

Reply:
I see that we add the `traceId` and `spanId` to context in this file (in `xhrProxy.beforeSend` and `fetchProxy.beforeSend`), so if we want to unset them inside the tracer, maybe we also want to set them there?

Reply:
good point, tracing methods already have access to context so moving the set logic here seems good to me

Reply:
ok, I will do that.